packages/test/test-end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -356,7 +356,7 @@ describeNoCompat("Loader.request", (getTestObjectProvider, apis) => {
 		);
 	});
 
-	it("requestResolvedObjectFromContainer can handle requests with headers", async () => {
+	it.only("requestResolvedObjectFromContainer can handle requests with headers", async () => {
```

Comment:
This unit test fails because we are trying to call request on the `entryPoint` (which is a DataObject) instead of on the `Container`.
See https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/src/loader.ts#L280

Reply:
@anthony-murphy What is the expectation for this scenario? Should there be handling added to `TestSharedDataObject1.request(...)` to route from the root DataObject to another one? Or should the `requestResolvedObjectFromContainer` method not use `entryPoint` at all (since we are currently calling `request` on 2 different things)?

Reply:
the first probably. to goal of requestResolvedObjectFromContainer is to abstract away who serves the request, the caller just wants the request to be handled.